integration_tests/suite/test_mobile_callback.py

```diff
@@ -1331,6 +1582,31 @@ def test_workflow_apns_with_app_using_two_tokens(self):
             ),
         )
 
+        def assert_incoming_call_notification(request):
+            assert_that(
+                request,
+                has_entry(
+                    'aps',
+                    has_entry(
+                        'alert',
```

Comment:
Suggestion: you can reduce the amount of indentation with `request['aps']['alert']`